editor/editor_autoload_settings.cpp

```diff
@@ -854,6 +889,8 @@ EditorAutoloadSettings::EditorAutoloadSettings() {
 	autoload_add_path = memnew(LineEdit);
 	hbc->add_child(autoload_add_path);
 	autoload_add_path->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	autoload_add_path->set_clear_button_enabled(true);
+	autoload_add_path->set_placeholder(TTR(vformat(R"(Path is empty, pressing "%s" button will create a new script.)", TTR("Add"))));
```

Comment:
Maybe it can be shortened a bit, as it might overflow or force a quite wide LineEdit otherwise (including when translated with languages like French which are more verbose than English):

Reply:
```suggestion
	autoload_add_path->set_placeholder(TTR(vformat(R"(Set path or press "%s" button to create a script.)", TTR("Add"))));
```